gel/_internal/_tracked_list.py

```diff
@@ -282,9 +282,7 @@ def _ensure_snapshot(self) -> None:
             self._removed_items = []
 
     def __gel_replace_with_empty__(self) -> None:
-        self._items.clear()
-        self._added_items = None
-        self._removed_items = None
+        self.clear()
```

Comment:
Why is there a special `__gel_replace_with_empty__` method anyway?
Isn't that just `clear`?